staging/cse/windows/kubernetesfunc.ps1

```diff
@@ -129,9 +129,68 @@ function Write-KubeClusterConfig {
         Destination = "c:\k";
     }
 
+    Update-KubeletConfigArgsForDualStack -ClusterConfiguration $Global:ClusterConfiguration
+
     $Global:ClusterConfiguration | ConvertTo-Json -Depth 10 | Out-File -FilePath $global:KubeClusterConfigPath
 }
 
+# Update-KubeletConfigArgsForDualStack updates the $Global:ClusterConfiguration object
+# to append the --node-ip argument for kubelet if the cluster is dualstack and the
+# host is configured with an IPv4 and IPv6 on the interface using the default route.
+# If the host does not have both IPs it will raise erro WINDOWS_CSE_ERROR_DUAL_STACK_NO_EXACT_TWO_IPS
+function Update-KubeletConfigArgsForDualStack {

```

Comment:
Could you help to test this change with a private cse package and a agentbaker tag in RP to validate whether this works with both dualstack and non-dualstack?

1. Create a folder aks-window-cse and copy all files including folder in staging/cse/windows to this folder
2. Use `zip -j aks-windows-cse-scripts-v1.0.22.zip aks-windows-cse/*` (Linux command) to create a test cse script pacakge. NOTE: Please not reuse the current cse script package version since the cache in AKS Windows image will be used instead of downloading it
3. Upload this cse package to a storage container and assure that it can be downloaded with anonymous permission
4. Update `parts/windows/kuberneteswindowssetup.ps1` in your test branch to set `CSEScriptsPackageUrl` to the full url to your test cse package in the storage account
5. Run `make generate` and commit it in your test branch
6. Create a test tag **in your forked repo**, for exmaple, ` v1.20230216.6` (The latest absvc tag in RP is `v0.20230216.5`)
7. Update absvc tag in RP and test both